tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -896,6 +896,8 @@ where
             {
                 let mut writer = self.make_writer.make_writer_for(event.metadata());
                 let _ = io::Write::write_all(&mut writer, buf.as_bytes());
```

Comment:
hmm, i wonder if we also want to print something if an I/O error occurs...

Reply:
Sure, I can add something here as well

Reply:
Addressed in my second commit